js/build.js

```
@@ -507,24 +511,27 @@
         if (updatedIndex !== -1) {
           this.pendingUpdates.updated.splice(updatedIndex, 1);
         }
 
         // Finally, add to deleted if not already there and not in inserted
-        if (!this.pendingUpdates.deleted.includes(deletion.id)) {
-          this.pendingUpdates.deleted.push(deletion.id);
+        if (!this.pendingUpdates.deleted.includes(deletion)) {
+          this.pendingUpdates.deleted.push(deletion);
         }
+
+        const deletedEntriesKey = 'deleted-entries-' + this.rows[0].dataSourceId;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)